bb_wrapper/wrapper/bb.py

```diff
@@ -78,9 +91,49 @@ def _auth(self):
             string de autenticação para o header
             Authorization
         """
-        return f"{self.__token_type} {self.__access_token}"
+        return f"{self.token_type} {self.access_token}"
+
+    @property
+    def access_token(self):
+        return self.__data.access_token
+
+    @access_token.setter
+    def access_token(self, value):
+        if not hasattr(self.__data, "access_token") or not self.__data.access_token:
+            self.__data.access_token = value
+
+    @property
+    def token_type(self):
+        return self.__data.token_type
 
-    def authenticate(self):
+    @token_type.setter
+    def token_type(self, value):
+        if not hasattr(self.__data, "token_type") or not self.__data.token_type:
```

Comment:
Curioso, não basta apenas uma dessas "expressões"?

Ou apenas `not hasattr(self.__data, "token_type")`

Ou apenas `not self.__data.token_type`?